OpenRA.Mods.Common/Widgets/ProductionPaletteWidget.cs

```diff
@@ -252,7 +252,8 @@ public override bool HandleKeyPress(KeyInput e)
 		public void RefreshIcons()
 		{
 			icons = new Dictionary<Rectangle, ProductionIcon>();
-			if (CurrentQueue == null)
+			var producer = CurrentQueue.MostLikelyProducer();
```

Comment:
got this in a playtest (in TD, right after surrendering), `CurrentQueue` needs a null check:

```
Exception of type `System.NullReferenceException`: Object reference not set to an instance of an object
  at OpenRA.Mods.Common.Widgets.ProductionPaletteWidget.RefreshIcons () [0x00012] in /home/oliver/devel/openra/git/OpenRA.Mods.Common/Widgets/ProductionPaletteWidget.cs:255 
  at OpenRA.Mods.Common.Widgets.ProductionPaletteWidget.set_CurrentQueue (OpenRA.Mods.Common.Traits.ProductionQueue value) [0x00009] in /home/oliver/devel/openra/git/OpenRA.Mods.Common/Widgets/ProductionPaletteWidget.cs:71 
  at OpenRA.Mods.Common.Widgets.ProductionPaletteWidget.Tick () [0x00032] in /home/oliver/devel/openra/git/OpenRA.Mods.Common/Widgets/ProductionPaletteWidget.cs:144
```